src/main/java/ch/njol/skript/entity/BoatChestData.java

```diff
@@ -114,15 +127,21 @@ public boolean isOfItemType(ItemType itemType) {
 		if (type == Material.OAK_CHEST_BOAT)
 			ordinal = 0;
 		else if (type == Material.SPRUCE_CHEST_BOAT)
-			ordinal = TreeSpecies.REDWOOD.ordinal();
+			ordinal = Boat.Type.SPRUCE.ordinal();
 		else if (type == Material.BIRCH_CHEST_BOAT)
-			ordinal = TreeSpecies.BIRCH.ordinal();
+			ordinal = Boat.Type.BIRCH.ordinal();
 		else if (type == Material.JUNGLE_CHEST_BOAT)
-			ordinal = TreeSpecies.JUNGLE.ordinal();
+			ordinal = Boat.Type.JUNGLE.ordinal();
 		else if (type == Material.ACACIA_CHEST_BOAT)
-			ordinal = TreeSpecies.ACACIA.ordinal();
+			ordinal = Boat.Type.ACACIA.ordinal();
 		else if (type == Material.DARK_OAK_CHEST_BOAT)
-			ordinal = TreeSpecies.DARK_OAK.ordinal();
+			ordinal = Boat.Type.DARK_OAK.ordinal();
+		else if (type == Material.CHERRY_CHEST_BOAT)
+			ordinal = Boat.Type.CHERRY.ordinal();
+		else if (type == Material.MANGROVE_CHEST_BOAT)
+			ordinal = Boat.Type.MANGROVE.ordinal();
+		else if (type == Material.BAMBOO_CHEST_RAFT)
+			ordinal = Boat.Type.BAMBOO.ordinal();
```

Comment:
Can all this be related to the enum so this doesn't have to be updated if any when Mojang adds another wood type?

Reply:
I will look into it